xchange-okcoin/src/main/java/org/knowm/xchange/okcoin/v3/dto/marketdata/OkexOrderBook.java

```diff
@@ -0,0 +1,75 @@
+package org.knowm.xchange.okcoin.v3.dto.marketdata;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import java.math.BigDecimal;
+import java.util.Arrays;
+
+public class OkexOrderBook {
+
+  private final String timestamp;
+  private final OkexOrderBookEntry[] bids;
+  private final OkexOrderBookEntry[] asks;
+
+  public OkexOrderBook(
+      @JsonProperty("timestamp") String timestamp,
+      @JsonProperty("bids") Object[][] bids,
+      @JsonProperty("asks") Object[][] asks) {
+
+    this.timestamp = timestamp;
+
+    if (bids != null && bids.length > 0) {
+      this.bids = new OkexOrderBookEntry[bids.length];
+      for (int i = 0; i < bids.length; i++) {
+        this.bids[i] = convertToBookEntry(bids[i]);
```

Comment:
if you use Jacskon property then you will not have to do this by hand

take a look how `@JsonFormat(shape = JsonFormat.Shape.ARRAY)` works

Reply:
Done.